pkg/agents/team2/actions.go

```diff
@@ -0,0 +1,64 @@
+package team2
+
+import (
+	"math/rand"
+
+	"github.com/SOMAS2021/SOMAS2021/pkg/utils/globalTypes/food"
+)
+
+func InitActionSpace() actionSpace {
+	//TODO: actionID might be removed in further versions
+	//index : 0 => disregard food
+	//index : 1 => satisfice with food
+	//index : 2 => satisfy with food
+	initialActionSpace := actionSpace{}
+	initialActionSpace.actionId = make([]int, 3)
+	for i := 0; i < 3; i++ {
+		initialActionSpace.actionId[i] = i
+	}
+	m := map[int]func(hp int) food.FoodType{
+		//actions based on the current hp level
+		initialActionSpace.actionId[0]: DisFood,
+		initialActionSpace.actionId[1]: Satisfice,
+		initialActionSpace.actionId[2]: Satisfy,
+	}
+	initialActionSpace.actionSet = m
+	return initialActionSpace
+}
+
+//Need to change this func when adding new actions
+
+func DisFood(hp int) food.FoodType {
+	return 0
+}
+
+func Satisfice(hp int) food.FoodType {
+	if hp <= 20 { //20 is the daily decay
```

Comment:
This is now no longer true: see https://github.com/SOMAS2021/SOMAS2021/wiki/Health for documentation.
In summary, your HP decay is dependent on your current hp.

Reply:
Yes, this was implemented before the introduction of Health. please ignore the comment for now as it still makes sense for an MVP. The logic will be substituted in the subsequent versions i.e. using the newly introduced health level instead of hp. Yes, again, I should've added TODO.